src/components/card/CreateNoteCard.css

```diff
@@ -0,0 +1,44 @@
+
+.create-note{
+    display:flex;
+    justify-content:center;
+    padding-top:5rem;
+}
+
+.create-note-container{
+    width:600px;
+    border-radius:8px;
+    box-shadow:rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
+}
+
+.note-title{
+    width:100%;
+    height:2.5rem;
+    border-radius:8px;
+    font-size:1.2rem;
+    outline:none;
+    border:none;
+    padding:0rem 0.5rem;
```

Comment:
Suggestion : You can use px for values less then 10px

Reply:
sure.